src/hooks/usePoolStats.ts

```diff
@@ -120,17 +130,15 @@ export const usePoolInfo = (address: string, signerAddress: string, tokenPrices:
     return date.toISOString();
   }, [address, signerAddress]);
 
-  const { data: tokensData, loading: tokensLoading } = useQuery<PoolTokensDataQuery, PoolTokensVar>(POOL_TOKENS_DATA_GQL, {
-    client: dexClient,
-    variables: { address },
-  });
+  const queryObj = getPoolTokensDataQuery(address);
+  const response = await graphqlRequest(httpClient, queryObj);
 
-  const { data: poolInfoData, loading: poolInfoLoading, refetch: refetchPoolInfo } = useQuery<PoolInfoQuery, PoolInfoVar>(POOL_INFO_GQL, {
-    client: dexClient,
-    variables: {
-      address, signerAddress, fromTime, toTime,
-    },
-  });
+  const { data: tokensData, loading: tokensLoading } = response.data;
+
+  const queryObj_ = getPoolInfoQuery(address, signerAddress, fromTime, toTime);
```

Comment:
name the queryObj variables so they're not the same - like poolInfoQry and poolTokensQry